edge-util/src/Microsoft.Azure.Devices.Edge.Util/Metrics.cs

```diff
@@ -92,6 +93,7 @@ public static void Count(MetricTags tags, CounterOptions options)
             MetricsCollector.ForEach(mroot =>
             {
                 mroot.Measure.Counter.Increment(options, tags);
+                mroot.Provider.Counter.Instance(options).Reset();
```

Comment:
We don't want this here I suppose. And add the `ResetOnReporting = true` property in all the places where we initialize a `CounterOptions` object.

Reply:
This was a good catch. Fixed.